api/v1alpha1/apidefinition_types.go

```diff
@@ -529,16 +528,17 @@ type ProxyTransport struct {
 	ProxyURL string `json:"proxy_url,omitempty"`
 }
 
+// CORS cors settings
 type CORS struct {
-	Enable             bool         `json:"enable"`
-	AllowedOrigins     []string     `json:"allowed_origins"`
-	AllowedMethods     []HttpMethod `json:"allowed_methods"`
-	AllowedHeaders     []string     `json:"allowed_headers"`
-	ExposedHeaders     []string     `json:"exposed_headers"`
-	AllowCredentials   bool         `json:"allow_credentials"`
-	MaxAge             int          `json:"max_age"`
-	OptionsPassthrough bool         `json:"options_passthrough"`
-	Debug              bool         `json:"debug"`
+	Enable             bool         `json:"enable,omitempty"`
```

Comment:
comments above each field will be propagated to oas3 schema.
This is useful for IDEs - as it forms documentation.

example: https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#define-the-api

<img width="740" alt="Screenshot 2020-11-12 at 21 25 55" src="https://user-images.githubusercontent.com/1465130/98998196-a8c26400-252d-11eb-8486-3d3d65d3a100.png">

Reply:
done dc16e57